src/api/api.controller.ts

```diff
@@ -165,6 +166,22 @@ export class ApiController {
         return await this.issuerService.revokeCredential(body.credential_exchange_id, body.publish);
     }
 
+    /**
+     * Deletes a credential for the issuer
+     */
+    @Delete('issuer/records/:credExId')
+    async deletedIssuedCredential(@Param('credExId') credExId: string): Promise<any> {
+        return await this.issuerService.deleteCredential(credExId);
+    }
+
+    /**
+     * Deletes a credential for the holder
```

Comment:
Maybe update the comment here to something like "Deletes a stored credential" or maybe the endpoint name to something like /credentials/stored or something. Or maybe a comment afterwards like "Note this is not the citizen". I just don't want anyone to think that holder here refers to a citizen in guardianship, but rather it's another entity in the system who just happens to have stored a credential and we can delete it here.